tests/basics.cpp

```diff
@@ -505,11 +505,11 @@ TEST_BOTH_FLOAT_AGNOSTIC(06_cast) {
     };
     const char *type_names[(int) VarType::Count]{
         "Void",   "Bool",  "Int8",   "UInt8",   "Int16",   "UInt16",  "Int32",
-        "UInt32", "Int64", "UInt64", "Pointer", "Float16", "Float32", "Float64"
+        "UInt32", "Int64", "UInt64", "Pointer", "", "Float16", "Float32", "Float64"
```

Comment:
This could be a bit confusing if the type ever gets printed in error messages or when debugging; is there a disadvantage to giving it a readable name like `"(BaseFloat)"`?

Reply:
Fixed, thanks!